tailwind.config.js

```diff
@@ -3,6 +3,13 @@ module.exports = {
   content: ["./*.{html,js}", "./src/**/*.{html,js}"],
   theme: {
     extend: {
+      height: {
+        'heightImage' : "26.13rem"
+      },
+      width: {
+        'widthImage': "45.09rem"
+
+      },
       lineHeight: {
         'extra-loose': '3.5rem',
       },
```

Comment:
We already have the extra-loose, you don't have to overwrite it. 
Example: if line-height is 60px, take 60/16 = x , take x *4 = y 
Replace the y where you write `extra-loose` and then x =3.5 in your case